skbio/metadata/intersection.pyx

```diff
@@ -0,0 +1,513 @@
+"""
```

Comment:
Please verify that this code is BSD compatible. From a google search, it looks like this code [originates](https://github.com/dnanexus/rseqc/blob/master/rseqc/lib/bx/intervals/intersection.pyx) from the rseqc project, which [appears](https://github.com/dnanexus/rseqc/blob/master/rseqc/lib/bx/intervals/intersection.pyx) to be GPLv3.

cc @gregcaporaso 
https://github.com/dnanexus/rseqc/blob/master/COPYING